src/download/linux.md

```diff
@@ -75,7 +75,7 @@ sudo dnf install prismlauncher-nightly
 ```
 
 </div>
-  
+  -->
```

Comment:
Would be nice to remove this if it's not needed anymore

Reply:
Changed, thank you!